WordPress/gutenberg
# Replace the renderBlockMenu prop with a Slot

Related #7192 

This PR replaces the `renderBlockMenu` Prop with Slot/Fill.

**Testing instructions**

 - Open the block menu and toggle the inspector (Show Block Settings)
 - It should work.
